src/routes/user/components/transaction_view/view.rs

```diff
@@ -219,23 +219,28 @@ pub fn format_transaction(
                                     .description
                                     .as_ref()
                                     .map(|val| format!(": {val}"));
+                                let fmt_description = move |
```

Comment:
nit: i dont think we technically need a move here since we do not capture any value